packages/manager/apps/pci-load-balancer/src/api/hook/useLoadBalancer.tsx

```diff
@@ -331,3 +333,47 @@ export const useLoadBalancerListeners = (
     ],
   );
 };
+
+export const useCreateLoadBalancer = ({
+  projectId,
+  flavor,
+  region,
+  floatingIp,
+  privateNetwork,
+  subnet,
+  gateways,
+  listeners,
+  name,
+  onSuccess,
+  onError,
+}: TCreateLoadBalancerParam & {
+  onError: (cause: Error) => void;
+  onSuccess: () => void;
+}) => {
+  const mutation = useMutation({
+    mutationFn: async () =>
+      createLoadBalancer({
+        projectId,
+        flavor,
+        region,
+        floatingIp,
+        privateNetwork,
+        subnet,
+        gateways,
+        listeners,
+        name,
+      }),
+    onError,
+    onSuccess: async () => {
+      // TDOO invalidate right query
```

Comment:
will it be done in another PR ?

Reply:
yep i forgot to invalidate the query, hopefully i left the TODO there, will fix it in the TUs task